hack/lib/constants.sh

```diff
@@ -43,6 +43,7 @@ readonly OS_CROSS_COMPILE_TARGETS=(
   cmd/openshift
   cmd/oc
   cmd/kubefed
+  cmd/openshift-diagnostics
```

Comment:
Hrm, confusing that this is just the binary used by diagnostics in pods.  It’s fine for now.

Reply:
> Hrm, confusing that this is just the binary used by diagnostics in pods. It’s fine for now.

Yeah.  I'm still open to different ways of structuring it as we push through 3.9.  For now, I need them out of openshift so I can snip `kubectl` dependencies and they likely belong together under a related suite of commands.

Reply:
@deads2k @smarterclayton got no problem with breaking this out, and don't much care about the name, but it needs to actually be included in the image which it isn't in 3.9-alpha.0... how can I make that happen?

Reply:
> @deads2k @smarterclayton got no problem with breaking this out, and don't much care about the name, but it needs to actually be included in the image which it isn't in 3.9-alpha.0... how can I make that happen?

I would try to work backwards from `images/origin/Dockerfile`.  We're installing the RPM, so if it isn't present, I suspect the RPM doesn't include it, which probably means fixing up origin.spec.